utilities/functions.py

```diff
@@ -53,3 +86,69 @@ def get_corners_from_contour(contour: np.ndarray, corner_number=4) -> None:
             upper = current
         i += 1
     return hull
+
+
+# The next two functions' `x` and `y` arguments are in camera coordinates.
+# The origin is the top left corner of the screen, with left and down being positive.
```

Comment:
left is positive? If 0,0 is at the top-left corner, then right is positive. Either this comment is wrong, or the code is.

Reply:
You're right, I've changed that.